modules/dns_nsupdate/dns_nsupdate_main.rb

```diff
@@ -13,22 +13,41 @@ def initialize(a_server = nil, a_ttl = nil)
     end
 
     def create_a_record(fqdn, ip)
-      do_create(fqdn, ip, "A")
+      case a_record_conflicts(fqdn, ip) #returns -1, 0, 1
+      when 1
+        raise(Proxy::Dns::Collision, "'#{fqdn} 'is already in use")
+      when 0 then
+        return nil
```

Comment:
No need to call 'return' here -- w/e expression evaluates to will be returned from the method.